python/CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v0.2.2] - October 9, 2024
+
+Loosen various dependency requirements
+
+Summary of changes:
+- [add missing double channel computation](https://github.com/sift-stack/sift/pull/92)
```

Comment:
Did you want to change this summary to just this version change?

Reply:
oh yes ty!

Reply:
copy-pasta...